src/main/rx/rx.c

```diff
@@ -72,6 +73,9 @@ const char rcChannelLetters[] = "AERT12345678abcdefgh";
 
 static uint16_t rssi = 0;                  // range: [0;1023]
 static timeUs_t lastMspRssiUpdateUs = 0;
+
+static pt1Filter_t rssiFilter;
```

Comment:
Variable should be renamed to be reflective of it's purpose as the filter is not used for true RSSI.

Reply:
Done